hbase_sql/src/main/java/com/splicemachine/olap/OlapServerMaster.java

```diff
@@ -89,58 +96,22 @@ private void run() throws Exception {
         // Initialize clients to ResourceManager and NodeManagers
         Configuration conf = HConfiguration.unwrapDelegate();
 
-        AMRMClientAsync.CallbackHandler allocListener = new AMRMClientAsync.CallbackHandler() {
-            @Override
-            public void onContainersCompleted(List<ContainerStatus> statuses) {
-            }
-
-            @Override
-            public void onContainersAllocated(List<Container> containers) {
-            }
-
-            @Override
-            public void onShutdownRequest() {
-                LOG.warn("Shutting down");
-                end.set(true);
-            }
-
-            @Override
-            public void onNodesUpdated(List<NodeReport> updatedNodes) {
-            }
-
-            @Override
-            public float getProgress() {
-                return 0;
-            }
-
-            @Override
-            public void onError(Throwable e) {
-                LOG.error("Unexpected error", e);
-                end.set(true);
-            }
-        };
-        AMRMClientAsync<AMRMClient.ContainerRequest> rmClient = AMRMClientAsync.createAMRMClientAsync(1000, allocListener);
-        rmClient.init(conf);
-        rmClient.start();
-
-        // Register with ResourceManager
-        rmClient.registerApplicationMaster(Utils.localHostName(), 0, "");
+        UserGroupInformation.isSecurityEnabled();
```

Comment:
The result is not used. Do we somehow rely on a side-effect of this call?

Reply:
Yes, it's mentioned in YARN documentation as a way to trigger Kerberos login, from https://hadoop.apache.org/docs/r2.7.3/hadoop-yarn/hadoop-yarn-site/YarnApplicationSecurity.html :

> This means you have a relative similar workflow across secure and insecure clusters.
> 1. Suring AM startup, log in to Kerberos. A call to UserGroupInformation.isSecurityEnabled() will trigger this operation.